src/content/en/2020/security.md

```diff
@@ -234,7 +234,7 @@ Looking at the different values for this attribute, we can clearly see that the
 )
 }}
 
-One website might have been a bit too enthusiastic about how long their site will be available over HTTPS and set a `max-age` attribute value that translates to 1,000,000,000,000,000 years. Ironically, browsers do not handle such a large value well, and actually disable HSTS for that site.
+One website might have been a bit too enthusiastic about how long their site will be available over HTTPS and set a `max-age` attribute value that translates to 1,000,000,000,000,000 years. Ironically, some browsers do not handle such a large value well, and actually disable HSTS for that site!
```

Comment:
BTW @tomvangoethem I found Chrome [does handle this](https://github.com/chromium/chromium/blob/99314be8152e688bafbbf9a615536bdbb289ea87/net/http/http_security_headers.cc#L24) and basically sets it to 1 year if over the 32-bit integer limit.

Couldn't find equivalent in Firefox so presume it overflows and, as you say, probably means no HSTS is set.

That's why I changed it to "some browsers"

Reply:
Good catch 👍